--- stock/resources/views/report/product_sell_report.blade.php
+++ live/resources/views/report/product_sell_report.blade.php
@@ -121,6 +121,7 @@
                                         <th>@lang('sale.customer_name')</th>
                                         <th>@lang('lang_v1.contact_id')</th>
                                         <th>@lang('lang_v1.contact_no')</th>
+                                        <th>@lang('business.email')</th>
                                         <th>@lang('sale.invoice_no')</th>
                                         <th>@lang('messages.date')</th>
                                         <th>@lang('sale.qty')</th>
@@ -134,7 +135,7 @@
                                 </thead>
                                 <tfoot>
                                     <tr class="bg-gray font-17 footer-total text-center">
-                                        <td colspan="9"><strong>@lang('sale.total'):</strong></td>
+                                        <td colspan="10"><strong>@lang('sale.total'):</strong></td>
                                         <td id="footer_total_sold"></td>
                                         <td></td>
                                         <td></td>
@@ -160,6 +161,7 @@
                                         <th>@lang('product.sku')</th>
                                         <th>@lang('sale.customer_name')</th>
                                         <th>@lang('lang_v1.contact_no')</th>
+                                        <th>@lang('business.email')</th>
                                         <th>@lang('sale.invoice_no')</th>
                                         <th>@lang('messages.date')</th>
                                         <th>@lang('lang_v1.purchase_ref_no')</th>
